<launch>
    <node pkg="mars-quadcopter" type="turtle_tf_broadcaster.py" name="turtle_tf_broadcaster" output="screen"></node>
</launch>
